### ISSUE TITLE
Hung on start (post a fresh installation

### ISSUE THREAD

**Issue by GH4TP** [files-community/Files#14063]
### Description

I installed both Files and Files Preview from the installers at https://files.community/ (not through the MS Store). Repeated attempts at getting both to launch after installation (including after rebooting) resulting in Files hanging.

### Steps To Reproduce

Install (using classic installer)
Launch

### Requirements

I guess I expected the application to start

### Files Version

Stable

### Windows Version

10.0.22635 Build 22635

### Log File

[debug.log](https://github.com/files-community/Files/files/13462850/debug.log)

**Comment by yaira2** [comment]
@GH4TP is this also an issue in the preview version?

**Comment by GH4TP** [comment]
I don't know @yaira2 as I have only tried the two installer versions currently available through the https://files.community/ website

**Comment by yaira2** [comment]
Never mind, I just noticed in the description that you already tried the preview version.

**Comment by yaira2** [comment]
Is the app also crashing?

**Comment by GH4TP** [comment]
I have always killed the process with Task Manager i.e. it may crash eventually but I've not experienced it

**Comment by GH4TP** [comment]
This is what I see post installation of 3.0.8.0
![image](https://github.com/files-community/Files/assets/38799710/afbf9ab8-3bb9-420e-8d49-08230d595d3c)

**Comment by yaira2** [comment]
Does it work on subsequent launches?

**Comment by GH4TP** [comment]
No. I get the same behaviour. I've tried rebooting and also get the same behaviour.

**Comment by GH4TP** [comment]
It does crash in Safe mode, though

**Comment by GH4TP** [comment]
Hi,
My sincere appologies. The problem with Files not running was caused by Bitdefender. Although there was nothing to indicate it was blocking Files (no notifications) I eventually turned everything off, reinstalled, and boom, the app started. So, I am sorry for wasting your time.
Warm regards
Tim

**Comment by Josh65-2201** [comment]
Thanks for the update.
